src/services/CertificateGenerationService.ts

```diff
@@ -1381,6 +1382,23 @@ class CertificateGenerationService {
       )
     );
   }
+
+  /**
+   * Sorts required required standards if present firstly by sectionNumber and then by rsNumber and then returns it
+   * @param requiredStandards - the requiredStandards collection to sort
+   * @returns - the sorted requiredStandards collection
+   */
+  private sortRequiredStandards = (requiredStandards: IRequiredStandard[] | undefined) : IRequiredStandard[] | undefined => {
+    if(!requiredStandards)
```

Comment:
Can you add the braces for the return here as per linting recommendation please.